tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/OffloadIndexBlockBuilder.java

```diff
@@ -71,7 +71,7 @@
     /**
      * Construct OffloadIndex from an InputStream.
      */
-    OffloadIndexBlock fromStream(InputStream is) throws IOException;
+    OffloadIndexBlock indexFromStream(InputStream is) throws IOException;
```

Comment:
Because now we have two different method with similar behavior, it's better to have a distinguish

``` 
   @Override
    public OffloadIndexBlock indexFromStream(InputStream is) throws IOException {
        return OffloadIndexBlockImpl.get(is);
    }

    @Override
    public StreamingOffloadIndexBlock streamingIndexFromStream(InputStream is) throws IOException {
        return StreamingOffloadIndexBlockImpl.get(is);
    }
```

Reply:
Sorry, I can't get the point here. Do you mean the method name will duplicated?

Reply:
> Sorry, I can't get the point here. Do you mean the method name will duplicated?

Yes